src/pypgstac/src/pypgstac/pypgstac.py

```diff
@@ -119,6 +119,121 @@ def loadextensions(self) -> None:
                 except Exception:
                     pass
 
+    def load_queryables(
+        self,
+        file: str,
+        collection_ids: Optional[str] = None,
```

Comment:
it seems that `fire` can understand the `list` type of so in theory you could pass `pypgstac load_queryables ... --collections_ids=["collection1", "collection2"]`

ref: https://github.com/google/python-fire/blob/master/docs/guide.md#argument-parsing